### ISSUE TITLE
Delete in destination orderInput by double click

### ISSUE THREAD

**Issue by tvedebrink** [Yang-Tang/shinyjqui#65]
Thanks for a great package @Yang-Tang 
Would it be possible to delete elements in the "destination" `orderInput` by double click or similar?
Cheers,
Torben

**Comment by graemediack** [comment]
I echo, great package and this would be a cool feature. But also, without this feature how do I code it so that I can remove items from destinations dragged by mistake, whilst still keeping the source as an unlimited supply of objects (as_source=TRUE)?

I'm using this UI logic currently, with a 'bin' destination, but it's a bit clunky so I'm hoping I'm missing some better way of doing it

```
orderInput('source', 'Source', items = month.abb, as_source = TRUE, connect = c('dest')), # SOURCE
orderInput('bin', 'Delete', items = NULL, placeholder = 'Delete here...'), # BIN
orderInput('dest', 'Dest', items = NULL, placeholder = 'Drag items here...',connect = c('bin','dest')), # DESTINATION
```

Thanks!

Graeme

**Comment by Yang-Tang** [comment]
Hi @tvedebrink @graemediack thanks for suggestion! I found someone has already figured out a solution:
https://community.rstudio.com/t/customizing-shinyjqui-package/48140/4
I have included this feature (4cdd1da2ba9cb706e2551bd64266b4184b0bdbf6) in the package. With this new GitHub version, you can delete any destination items by dragging them back.
